runtime.go

```diff
@@ -63,6 +64,14 @@ type _runtime struct {
 
 func (self *_runtime) enterScope(scope *_scope) {
 	scope.outer = self.scope
+	if self.scope != nil {
+		scope.depth = self.scope.depth + 1
+	}
+
+	if self.stackLimit != 0 && scope.depth >= self.stackLimit {
+		panic(self.panicRangeError("RangeError: Maximum call stack size exceeded"))
```

Comment:
RangeError knows it's a RangeError so we shouldn't need the message prefix here.

I can see you've followed some other examples of its use, but I believe they are wrong too.